### PR TITLE
Event bug fixes and improvements

### PR DESCRIPTION
**Pull request by Axel-Jacobsen**
Takes care of #635, #637, `https://github.com/ubyssey/ubyssey-dispatch-theme/issues/289`,

### CODE REVIEW

File: dispatch/apps/events/forms.py [hunk 1]
```diff
@@ -19,19 +19,21 @@ class Meta:
...
+            'ticket_url',
             'is_submission',
             'submitter_email',
             'submitter_phone'
         ]
 
         widgets = {
             'title': TextInput(attrs={'placeholder': 'Title'}),
-            'description': TextInput(attrs={'placeholder': 'Description'}),
+            'description': Textarea(attrs={'placeholder': 'Description'}),
```
Comment (Axel-Jacobsen) [inline dispatch/apps/events/forms.py:33]: Takes care of #637

File: dispatch/apps/events/forms.py [hunk 2]
```diff
@@ -1,4 +1,6 @@
-from django.forms import ModelForm, DateTimeField, CharField, TextInput
+from datetime import datetime
+
+from django.forms import ModelForm, DateTimeField, CharField, TextInput, Textarea
 from dispatch.apps.events.models import Event
```
Comment (psiemens) [inline dispatch/apps/events/forms.py]: Add empty line between django and dispatch imports

File: dispatch/apps/events/forms.py (hunk 1 above)
Comment (psiemens) [reply dispatch/apps/events/forms.py:33]: Nice 👍

**Review by psiemens — CHANGES_REQUESTED** [review]

**Comment by psiemens** [comment]
There was just one small change required, so I went ahead and fixed it before merging. Good stuff -- the date validation is something I hadn't thought of

**Review by psiemens — APPROVED** [review]

**Pull request merged**